koyo-lib/blueprints/standard/app-name-here/config.rkt

```diff
@@ -69,6 +69,8 @@
 (define-option session-secret-key
   (or session-secret-key (make-secret-salt/file session-secret-key-path)))
 
+(define-option browsersync #:default #t)
```

Comment:
Default values should always be strings, because we don't do any coercion of the values we get from the environment. So this would have to be

```racket
(define-option browsersync #:default "x"
   (equal? browsersync "x"))
```

That is, `browsersync` is `#t` when the environment variable contains the value `"x"`.

That said, my preference would be to drop the default and simply add `APP_NAME_HERE_BROWSERSYNC=x` to the example `.env` file.

```racket
(define-option browsersync
  (equal? browsersync "x"))
```

This would be in line with how the `config` and `profile` options work.